src/layouts/Application.astro

```diff
@@ -41,6 +43,21 @@ import logoLargeWhite from '../images/svgs/pointless-logo-large--white.svg';
 import iconX from '../images/svgs/x.svg?raw';
 import '../styles/global.css';
 ---
+{(() => {
```

Comment:
This was really weird and it even led me to [a recent bug report by Henry](https://github.com/withastro/compiler/issues/1009)! Moving this code outside the frontmatter fixed it not wanting to access the env variables.